README.md

```diff
@@ -14,6 +14,7 @@ This repository contains the Dafny formalization of Cedar and infrastructure for
 To build the Dafny formalization and proofs:
 
 * Install Dafny 4.0, following the instructions [here](https://github.com/dafny-lang/dafny/wiki/INSTALL). Our proofs expect Z3 version 4.12.1, so if you have another copy of Z3 installed locally, you may need to adjust your PATH.
+* Note that our Makefile also expects the .NET SDK and `dotnet` executable, version 6.0.x, available on your PATH. You can download the .NET SDK 6.0 from [here](https://dotnet.microsoft.com/en-us/download/dotnet/6.0).
```

Comment:
Yes but the linked installation instructions don't specify that you need the .NET SDK (not just the .NET runtime) and don't specify that the `dotnet` executable must be on your PATH.  Although possibly we should just submit that patch to the linked instructions rather than here